### PR TITLE
update auto-update cfg & add missing files of numeral.js

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #9479 
@pvnr0082t please help me review this pull request, thanks!
update npmFileMap in package.json to grab languages directory automatically and add the languages files for the last added version.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@kennynaoh maybe also add the language files for the last added version?

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have also add the languages files for the last added version.
Please help me review it again.
Thanks!

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Should we need to add `language.min.js` in this case?
Please help me confirm that.
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh why not?

**Comment by kennynaoh** [comment]
OK, I will add it, thanks!

**Comment by kennynaoh** [comment]
@pvnr0082t 
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
BTW, remember to assign PR to your reviewer. @kennynaoh

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated the commit message.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

File: ajax/libs/numeral.js/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     {
       "basePath": "min",
       "files": [
-        "numeral.min.js"
+        "+(numeral|languages).*",
+        "languages/*"
```
Comment (PeterDaveHello) [inline ajax/libs/numeral.js/package.json]: Should use `"**/*"` here.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have fixed the fileMap.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**